ckanext/canada/tables/briefingt.yaml

```diff
@@ -101,6 +116,9 @@ resources:
       en: This field will display the person to whom the briefing note is addressed.
     obligation: Mandatory
     excel_required: true
+    form_required: true
+    form_attrs:
+      style: 'width:559.87px;'
```

Comment:
@wardi are you ok with width defined in pixels? I prefer percentage (if needed) so that it can adapt to screen size